io/cmd.go

```diff
@@ -243,7 +243,7 @@ func (config *Command) GetCmd() (cmd *exec.Cmd) {
 	if config.CmdName != "" {
 		cmdStr = append(cmdStr, config.CmdName)
 	}
-	if config.CmdArgs != nil && len(config.CmdArgs) > 0 {
```

Comment:
Why remove `config.CmdArgs != nil`?

Reply:
The new Go linter threw an error for this issue. `len()` on nil is 0 as well as an empty slice, thus the nil check is redundant.